tooling/webpack/graphqlLoader.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(source) {
+  this.cacheable()
+  this.value = source
+  const sanitizedSource = source.replace(/`/g, "\\`")
+  return `export default \`${sanitizedSource}\``
+}
```

Comment:
It would be worth investigating some other implementations: https://github.com/samsarahq/graphql-loader